tests/test_browser.py

```diff
@@ -3348,6 +3348,11 @@ def test_modularize_network_error(self):
     create_test_file('a.html', '''
       <script src="a.out.js"></script>
       <script>
+        if (typeof window === 'object' && window) {
```

Comment:
Is the `&& window` needed?

Reply:
I also think it is not needed. window is either valid or undefined. And typeof undefined is 'undefined'.
(typeof null is 'object', which is ... special ... but doesn't apply here as window will never be null).